package proto_ff;

import "yd_fieldoptions.proto";


message E_GuildConstant
{
	optional int32 m_constantid = 1[(yd_fieldoptions.field_cname) = "配置id"];
	optional int32 m_setitem = 2[(yd_fieldoptions.field_cname) = "创建公会需要道具ID"];
	optional int32 m_nameitem = 3[(yd_fieldoptions.field_cname) = "公会改名需要道具ID"];
	optional int32 m_editrequirelv = 4[(yd_fieldoptions.field_cname) = "修改公会公告要求公告等级"];
	optional int32 m_giftcd = 5[(yd_fieldoptions.field_cname) = "红包清理时间"];
	optional int32 m_dismiss = 6[(yd_fieldoptions.field_cname) = "解散公会时间"];
	optional int32 m_transferguild = 7[(yd_fieldoptions.field_cname) = "代理会长可申请时间"];
	optional int32 m_territory = 8[(yd_fieldoptions.field_cname) = "公会领地场景id"];
	optional int32 m_autominlv = 9[(yd_fieldoptions.field_cname) = "自动审批最低等级"];
	optional int32 m_autominbp = 10[(yd_fieldoptions.field_cname) = "自动审批最低战力"];
	optional int32 m_donatetimes = 11[(yd_fieldoptions.field_cname) = "捐款次数"];
	optional int32 m_packetmaxd = 12[(yd_fieldoptions.field_cname) = "红包绑钻每日限制"];
	optional int32 m_packetmaxbd = 13[(yd_fieldoptions.field_cname) = "红包魔晶每日限制"];
	optional int32 m_packetmaxsend = 14[(yd_fieldoptions.field_cname) = "自费红包限制"];
	optional int32 m_packetvip = 15[(yd_fieldoptions.field_cname) = "红包加成的vip等级"];
	optional int32 m_packetadd = 16[(yd_fieldoptions.field_cname) = "红包加成比例"];
	optional int32 m_prestigetasonekeyvip = 17[(yd_fieldoptions.field_cname) = "声望任务一键vip"];
	optional int32 m_prestigetasonekeylv = 18[(yd_fieldoptions.field_cname) = "声望任务一键vip"];
	optional int32 m_prestigetaskprivage = 19[(yd_fieldoptions.field_cname) = "声望任务特权id"];
	optional int32 m_prestigetaskflash = 20[(yd_fieldoptions.field_cname) = "声望任务免费刷新次数"];
	optional int32 m_prestigetaskflashprice = 21[(yd_fieldoptions.field_cname) = "声望任务付费刷新价格"];
	optional int32 m_prestigetasktimes = 22[(yd_fieldoptions.field_cname) = "声望任务每日次数"];
}


message Sheet_GuildConstant
{
	repeated E_GuildConstant E_GuildConstant_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_GuildLevel
{
	optional int32 m_guildlevel = 1[(yd_fieldoptions.field_cname) = "公会等级"];
	optional int32 m_guildlevel_exp = 2[(yd_fieldoptions.field_cname) = "公会升级经验"];
	optional int32 m_guildlevel_member = 3[(yd_fieldoptions.field_cname) = "公会成员总人数"];
	optional int32 m_guildlevel_vp = 4[(yd_fieldoptions.field_cname) = "公会副会长人数"];
	optional int32 m_guildlevel_np = 5[(yd_fieldoptions.field_cname) = "公会长老数"];
	optional int32 m_guildlevel_g = 6[(yd_fieldoptions.field_cname) = "护法人数"];
}


message Sheet_GuildLevel
{
	repeated E_GuildLevel E_GuildLevel_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_GuildPosition
{
	optional int32 m_positionid = 1[(yd_fieldoptions.field_cname) = "职位ID"];
	optional int32 m_admitnewcomer = 2[(yd_fieldoptions.field_cname) = "接收新成员"];
	optional int32 m_appointandremovevp = 3[(yd_fieldoptions.field_cname) = "任命/解除副会长"];
	optional int32 m_appointandremoveelder = 4[(yd_fieldoptions.field_cname) = "任命/解除长老"];
	optional int32 m_appointandremoveg = 5[(yd_fieldoptions.field_cname) = "任命/解除护法"];
	optional int32 m_appointandremoveelite = 6[(yd_fieldoptions.field_cname) = "任命/解除精英"];
	optional int32 m_expelvp = 7[(yd_fieldoptions.field_cname) = "踢出副会长"];
	optional int32 m_expelelder = 8[(yd_fieldoptions.field_cname) = "踢出长老"];
	optional int32 m_expelg = 9[(yd_fieldoptions.field_cname) = "踢出护法"];
	optional int32 m_expelmass = 10[(yd_fieldoptions.field_cname) = "踢出普通成员"];
	optional int32 m_announcementediting = 11[(yd_fieldoptions.field_cname) = "编辑公告"];
	optional int32 m_rname = 12[(yd_fieldoptions.field_cname) = "修改名字"];
	optional int32 m_joincondition = 13[(yd_fieldoptions.field_cname) = "设置加入方式"];
	optional int32 m_guildactivity = 14[(yd_fieldoptions.field_cname) = "开启公会玩法"];
	optional int32 m_dissolveassignguild = 15[(yd_fieldoptions.field_cname) = "解散/转让公会"];
	optional int32 m_changename = 16[(yd_fieldoptions.field_cname) = "改名权限"];
	optional int32 m_recruitchat = 17[(yd_fieldoptions.field_cname) = "发布招贤纳士"];
	optional int32 m_guildpacket = 18[(yd_fieldoptions.field_cname) = "发送公会玩法红包"];
}


message Sheet_GuildPosition
{
	repeated E_GuildPosition E_GuildPosition_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_GuildSalaryRewardDesc
{
	optional int32 m_item = 1[(yd_fieldoptions.field_cname) = "物品"];
	optional int32 m_num = 2[(yd_fieldoptions.field_cname) = "数量"];
}

message E_GuildSalary
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "任务类型"];
	optional int32 m_typearg = 3[(yd_fieldoptions.field_cname) = "类型参数"];
	optional int32 m_times = 4[(yd_fieldoptions.field_cname) = "次数参数"];
	repeated E_GuildSalaryRewardDesc m_reward = 5[(yd_fieldoptions.field_cname) = "奖励", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_GuildSalary
{
	repeated E_GuildSalary E_GuildSalary_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_GuildDonateRewardDesc
{
	optional int32 m_item = 1[(yd_fieldoptions.field_cname) = "物品"];
	optional int32 m_num = 2[(yd_fieldoptions.field_cname) = "数量"];
}

message E_GuildDonate
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 m_costitem = 2[(yd_fieldoptions.field_cname) = "花费物品id"];
	optional int32 m_costnum = 3[(yd_fieldoptions.field_cname) = "花费物品数量"];
	repeated E_GuildDonateRewardDesc m_reward = 4[(yd_fieldoptions.field_cname) = "奖励", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_GuildDonate
{
	repeated E_GuildDonate E_GuildDonate_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message E_GuildPacket
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "红包ID"];
	optional int32 m_belongtype = 2[(yd_fieldoptions.field_cname) = "红包归属类型"];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "红包触发类型"];
	optional int32 m_typearg = 4[(yd_fieldoptions.field_cname) = "类型参数"];
	optional int32 m_itemid = 5[(yd_fieldoptions.field_cname) = "红包领取类型"];
	optional int32 m_itemnum = 6[(yd_fieldoptions.field_cname) = "红包金额"];
	optional int32 m_num = 7[(yd_fieldoptions.field_cname) = "红包份数"];
	optional int32 m_pre = 8[(yd_fieldoptions.field_cname) = "前置红包"];
	optional int32 m_daily = 9[(yd_fieldoptions.field_cname) = "每日限制触发次数"];
}


message Sheet_GuildPacket
{
	repeated E_GuildPacket E_GuildPacket_List = 1[(yd_fieldoptions.field_arysize)=80];
}

message E_GuildPrestigetask
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_level = 2[(yd_fieldoptions.field_cname) = "任务分级"];
	optional int32 m_weight = 3[(yd_fieldoptions.field_cname) = "权重"];
	optional int32 m_taskpool = 4[(yd_fieldoptions.field_cname) = "随机的任务池id"];
	optional int32 m_reward = 5[(yd_fieldoptions.field_cname) = "任务奖励类型id"];
}


message Sheet_GuildPrestigetask
{
	repeated E_GuildPrestigetask E_GuildPrestigetask_List = 1[(yd_fieldoptions.field_arysize)=20];
}
